src/common/options.cc

```diff
@@ -1149,9 +1149,8 @@ std::vector<Option> get_global_options() {
     .set_default(100ul << 20)
     .set_description(""),
 
-    Option("mon_mgr_proxy_client_bytes_ratio", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
-    .set_default(.3)
-    .set_description(""),
+    Option("mon_mgr_proxy_client_bytes_ratio", Option::TYPE_FLOAT, Option::LEVEL_DEV)
+    .set_default(.3),
```

Comment:
can we copy the comment of 
> ratio of mon_client_bytes that can be consumed by proxied mgr commands before we error out to client

from `legacy_config_opts.h` here?

i understand that this option is not a user-facing knob. but the comment might help developers. what do you think?

Reply:
Done